README.md

```diff
@@ -1,2 +1,54 @@
-# go-http-message-signatures
-Go implementation of the HTTP Message Signatures RFC
+# Go HTTP Message Signatures
+
+Go implementation of the [draft HTTP Message Signatures RFC](https://www.ietf.org/archive/id/draft-cavage-http-signatures-12.txt). 
+
+**Please note the version of the draft this is based on as there are varying versions which are incompatible.**
+
+## Usage
+
+`import "github.com/form3tech-oss/go-http-message-signatures"
```

Comment:
nit: I think it is more common to write `go get ... ` in readme instead of `import`